app/controllers/navigation/Navigator.scala

```diff
@@ -69,7 +69,7 @@ object Navigator {
   }
 
   val simplified: PartialFunction[DeclarationPage, Mode => Call] = {
-    case BorderTransport        => controllers.declaration.routes.WarehouseDetailsController.displayPage
+    case BorderTransport        => controllers.declaration.routes.InlandTransportDetailsController.displayPage
```

Comment:
Is this page not for all journeys? Is it just for Simplified?

Reply:
Not page, I mean this redirect :)

Reply:
It is for all journeys